yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -315,6 +315,29 @@ Internet-Draft                 YANG Semver                 November 2021
       -  "_non_compatible" - the change represents a non-backwards-
          compatible change
 
+   The _COMPAT modifier string is "sticky".  Once a revision of a module
+   has a modifier in the revision label, then all descendants of that
+   revision will also have a modifier.  The modifier can change from
+   _compatible to _non_compatible in a descendant revision, but the
+   modifier MUST NOT change from _non_compatible to _compatible and MUST
+   NOT be removed.  The persistence of the _non_compatible modifier
+   ensures that comparisions of revision labels do not give the false
+   impression of compatibility between two potentially non-compatible
+   revisions.  If _non_compatible was removed, for example between
```

Comment:
If the "_non_compatible" modifier ...

I think that we should probably quote all instances of "_compatible" and "_non_compatible" in this paragraph.

Reply:
done.  Also added double quotes to a number of other revision labels around the doc.